server/core/templates/login.html

```diff
@@ -15,24 +15,6 @@
 			required=true
 		/>
 
-		<input
-			class="input-hidden"
-			id="password"
-			name="password"
```

Comment:
@weijiangan Please don't remove these, they are used by password managers, so that when you autofill the username it also completes the PW + TOTP in the background, so that it's really painless to login